shippers/schemas/Orders.json

```diff
@@ -1104,69 +1104,60 @@
-                      "pickup": {
-                        "name": "Rob's Lot",
-                        "address": "4323 N 78th St, Scottsdale, AZ 85251, USA",
-                        "geolocation": {
-                          "coordinates": [
-                            -111.911547,
-                            33.4993709
-                          ]
-                        },
-                        "type": "business"
+                        "type": "string"
                       },
-                      "dropoff": {
-                        "name": "Jerry's House",
-                        "address": "4230 E Sells Dr, Phoenix, AZ 85018, USA",
+                      "pickup": {
+                        "address": "string",
                         "geolocation": {
                           "coordinates": [
-                            -111.9897083,
-                            33.5017089
-                          ]
+                            0
+                          ],
+                          "type": "string"
                         },
-                        "type": "personal"
+                        "type": "string"
                       }
                     },
                     "vehicle": {
-                      "vin": "JTJHY7AX8B4079281"
+                      "color": "Blue",
+                      "isOperational": true,
+                      "isOverSized": false,
+                      "make": "BMW",
+                      "model": "M3",
+                      "notes": "She's a beauty.",
+                      "odometer": 10000,
+                      "stockNumber": "1234",
+                      "truckSpecifications": {
+                        "rearAxle": "string"
+                      },
+                      "type": "string",
+                      "vehicleMarket": "LIGHT",
+                      "vehicleMeasurements": {
+                        "curbWeight": 0,
+                        "grossVehicleWeightRating": 0,
+                        "height": 0,
+                        "length": 0,
+                        "width": 0
+                      },
+                      "vin": "2C3KA53G68H269241",
+                      "year": 2015
                     }
                   }
                 ]
-              }
+              },
+              "Minimal": "// Minimal information required for quote request.  Fare name optional. {     \"type\": \"BASIC\",     \"fare\": {         \"name\": \"Gold\"     },     \"vehicleTransferOrders\": [         {             \"directions\": {                 \"pickup\": {                     \"address\": \"2401 S 15th Ave, Phoenix, AZ 85007, USA\"                 },                 \"dropoff\": {                     \"address\": \"154 Chester Drive, Yonkers, NY 10710, USA\"                     }             },             \"vehicle\": {                 \"vin\": \"1J4GW58SXXC527112\",                 \"isOperational\": true             }         }     ] }"
```

Comment:
need to remove the `// Minimal information required for quote request.  Fare name optional.` for valid json. Also, no need to escape the quotes.

If you want to convey this type of info, you can use the description.